src/vsr/superblock.zig

```diff
@@ -130,16 +127,24 @@ pub const SuperBlockHeader = extern struct {
         /// The view number of the replica.
         view: u32,
 
-        reserved: [8]u8 = [_]u8{0} ** 8,
+        /// Identity of the replica, part of VSR configuration.
+        replica: u8,
+
+        /// Number of replicas (determines sizes of the quorums), part of VSR configuration.
+        replica_count: u8,
+
+        reserved: [6]u8 = [_]u8{0} ** 6,
```

Comment:
@sentientwaffle any tips on byte-golfing here? I did the laziest possible thing of:

* extending `copy` to `u16` to avoid shifting everything
* for VSRState, just eat into two reserved bytes (which sadly requires me to replace u64 loops with u8 ones)

Reply:
Got myself overhyped that I can just write `version: u24` in Zig only to be let down by the compiler: 

```
error: extern structs cannot contain fields of type 'u24'
```

Reply:
Ah, forgot about that. :disappointed: 
Maybe just change `copy` to `u16` with a TODO that it can be reduced? Or just replace the old replica index with a single `u8` padding byte.

Reply:
made it u16 back: we already have reserved bytes, don't want to have two discountiguous reservations.